### ISSUE TITLE
six 1.3 doesn't have PY2

### ISSUE THREAD

**Issue by TomasTomecek** [goldmann/docker-scripts#16]
Version:

python-six-1.3.0-4.el7.noarch

```
2015-05-13T16:06:17.324907639Z   File "/usr/lib/python2.7/site-packages/docker_scripts/squash.py", line 22, in <module>
2015-05-13T16:06:17.335738340Z     if six.PY2:
2015-05-13T16:06:17.335738340Z AttributeError: 'module' object has no attribute 'PY2'
```

**Comment by goldmann** [comment]
Ech, what an old version we have there... :/

**Comment by goldmann** [comment]
Fixed in 0.3.6.
